python/cudf/cudf/_lib/aggregation.pyx

```diff
@@ -403,46 +177,26 @@ cdef class Aggregation:
     def all(cls):
         return cls(pylibcudf.aggregation.all())
 
+    # Rolling aggregations
+    @classmethod
+    def from_udf(cls, op, *args, **kwargs):
+        # Handling UDF type
+        nb_type = numpy_support.from_dtype(kwargs['dtype'])
```

Comment:
Nit: Would be since if `dtype` was explicitly in the signature.

Reply:
Agreed. There's a lot of rewriting of the Cython layer that we should do once pylibcudf is completed.